03-setup-ingress-controller/ambassador.md

```diff
@@ -184,9 +198,97 @@ IP                 ID                                      Name
 
 In the next step, you will learn how to create the `Host` CRDs which tell `Ambassador` how to expose backend hosts (services) to the outside world.
 
-## Step 2 - Defining the Hosts for Ambassador Edge Stack
+## Step 2 - Defining the Listener for Ambassador Edge Stack
+
+The `Listener` CRD defines where, and how, Ambassador Edge Stack should listen for requests from the network (eg DO Load Balancer), and which Host definitions should be used to process those requests.
+
+**Note:**
+
+The `Listener`s are never created at the installation, the below steps will guide you how to do it.
+
+A typical `Listener` configuration looks like below:
+
+```yaml
+apiVersion: getambassador.io/v3alpha1
+kind: Listener
+metadata:
+  name: http-listener
+spec:
+  port: 8080
+  protocol: HTTPS
+  securityModel: SECURE
+  hostBinding:
+    namespace:
+      from: ALL
+```
+
+Explanations for the above configuration:
+
+- `port`: The network port on which Ambassador Edge Stack should listen.
+- `protocol`: The protocol Type on which Ambassador Edge Stack will use.
+- `protocolStack`: Allows configuring exactly which protocols will be layered together.
+- `securityModel`: Defines how the Listener will decide whether a request is secure or insecure.
+- `hostBinding`: Mechanism for determining which Hosts will be associated with this Listener.
+
+For more details, please visit the AES [Listener](https://www.getambassador.io/docs/edge-stack/2.1/topics/running/listener/) CRD official documentation.
+
+First, change directory (if not already) where you cloned the `Starter Kit` repository.
+
+```shell
+cd Kubernetes-Starter-Kit-Developers
+```
+
+Next, apply the manifest to create the `Listener`:
+
+```shell
+kubectl apply -f 03-setup-ingress-controller/assets/manifests/ambassador/ambassador_listener.yaml
+```
+
+**Observations and results:**
+
+Inspect the AES `Listener`:
 
-In a real world scenario each `host` maps to a `service`, so you need a way to tell `AES` about your intentions - meet the [Host](https://www.getambassador.io/docs/edge-stack/1.13/topics/running/host-crd/) CRD. The `Host` CRD can handle `TLS termination` automatically, by using `HTTP-01` ACME challenge to request `TLS certificates` from a well known `Certificate Authority` (like [Let's Encrypt](https://letsencrypt.org/)). Certificates creation and renewal happens automatically once you configure and enable this feature in the `Host` CRD.
+```shell
+kubectl describe listener.getambassador.io
+```
+
+The output looks similar to the following:
```

Comment:
I would add here a note like this:

The output looks similar to the following (notice the creation of host bindings via the `Spec.Host Binding` field):

Reply:
Thank you, updated the line